chapter_recurrent-modern/seq2seq_vn.md

```diff
@@ -201,7 +205,11 @@ The only difference is that we add a dense layer after the LSTM layers, where th
 The dense layer will predict the confidence score for each word.
 -->
 
-*dịch đoạn phía trên*
+Khi cài đặt bộ giải mã, chúng ta trực tiếp sử dụng trạng thái ẩn ở bước cuối cùng của bộ mã hoá làm trạng thái ẩn đầu tiên của bộ giải mã.
+Điều này đòi hỏi bộ mã hoá và bộ giải mã phải có cùng số tầng và số nút ẩn.
+Các bước tính toán lan truyền xuôi trong mô hình LSTM của bộ giải mã tương tự như trong bộ mã hoá.
```

Comment:
```suggestion
Các bước tính toán lượt truyền xuôi trong mô hình LSTM của bộ giải mã tương tự như trong bộ mã hoá.
```

Reply:
Em nghĩ chỗ này **LSTM forward calculation** tức là **tính toán lan truyền xuôi** chứ ạ. Em không hiểu ý anh, tại sao sửa **lan** thành **lượt** ạ ?

Reply:
ah vì mọi người thống nhất như vậy trong glossary, `propagation` mới dịch là `lan truyền`. Bạn có thể xem thảo luận cũ ở đây nhé https://github.com/aivivn/d2l-vn/pull/812/

Reply:
à vâng :') hôm qua em cũng tìm forward calculation nhưng không có